models/grading/fact_grade_status.sql

```diff
@@ -0,0 +1,38 @@
+select
+    ls.org as org,
+    ls.course_key as course_key,
+    ls.actor_id as actor_id,
+    courses.course_name as course_name,
+    courses.course_run as course_run,
+    state as state,
+    course_grade as course_grade,
+    case
+        when course_grade >= 0.9
+        then '90-100%'
+        when course_grade >= 0.8 and course_grade < 0.9
+        then '80-89%'
+        when course_grade >= 0.7 and course_grade < 0.8
+        then '70-79%'
+        when course_grade >= 0.6 and course_grade < 0.7
+        then '60-69%'
+        when course_grade >= 0.5 and course_grade < 0.6
+        then '50-59%'
+        when course_grade >= 0.4 and course_grade < 0.5
+        then '40-49%'
+        when course_grade >= 0.3 and course_grade < 0.4
+        then '30-39%'
+        when course_grade >= 0.2 and course_grade < 0.3
+        then '20-29%'
+        when course_grade >= 0.1 and course_grade < 0.2
+        then '10-19%'
+        else '0-9%'
+    end as grade_bucket
```

Comment:
Since we're using this in more than one model, I think it makes sense to move this into a macro.